crates/db_schema/src/impls/images.rs

```diff
@@ -67,3 +98,16 @@ impl RemoteImage {
     }
   }
 }
+
+impl ImageDetails {
+  pub(crate) async fn create(
+    conn: &mut AsyncPgConnection,
+    form: &ImageDetailsForm,
+  ) -> Result<usize, Error> {
+    insert_into(image_details::table)
+      .values(form)
+      .on_conflict_do_nothing()
+      .execute(conn)
```

Comment:
I learned the hard way that you can't call `on_conflict_do_nothing()` with `get_result`, otherwise it will return an error.